rero/rero-ils
# documents: improve holdings display performance

* adapts of APIs to allow angular to retrieve data
* fixes test of APIs

Co-Authored-by: Bertrand Zuchuat <bertrand.zuchuat@rero.ch>

## Dependency

- https://github.com/rero/rero-ils/pull/1591

## How to test
- run `npm run pack` on rero-ils-ui
- install `poetry run bootstrap -t /path/to/rero-ils-ui/build/rero-rero-ils-ui-0.10.0.tgz`
- data `poetry run setup -C`
- public interface: check document detail view >> get tab (paging)
- admin interface: check document holdings (paging)
- Test requests on items

## Code review check list
